src/orange/LevelStateAccessor.hh

```diff
@@ -7,14 +7,19 @@
 //---------------------------------------------------------------------------//
 #pragma once
 
+#include "corecel/Types.hh"
+
 #include "OrangeData.hh"
 #include "OrangeTypes.hh"
 
 namespace celeritas
 {
 //---------------------------------------------------------------------------//
 /*!
- * Access the 2D fields (i.e., {track slot, level}) of OrangeStateData.
+ * Access the 2D fields (i.e., {track slot, udepth}) of OrangeStateData.
+ *
+ * \internal The class name is from the legacy nomenclature of "level" meaning
```

Comment:
I don't think we should keep it this way long term. Add a \todo to rename to `UnivDepthStateAccessor`